ManualSetup/6_InsertData_TB_USER.sql

```diff
@@ -0,0 +1,15 @@
+INSERT INTO [TB_USER] (strAccountID, strPasswd, strSocNo, strEmail, strAuthority, PremiumExpire) VALUES
+(N'BitShift', CONVERT(varchar(13), 0x74657374), CONVERT(varchar(20), 0x), CONVERT(varchar(250), 0x), 1, CAST(N'2016-05-26T21:48:24.15' AS DateTime)),
```

Comment:
The following columns shouldn't be hex:
strPasswd (I can see a use, but it's also just placeholder passwords here for playing around)
strSocNo
strEmail

It also seems they don't handle empty strings well, though it probably still works. Either way, they shouldn't be hex.